frame/society/src/lib.rs

```diff
@@ -161,7 +161,7 @@ decl_storage! {
 		}): Vec<T::AccountId>;
 
 		/// The set of suspended members.
-		pub SuspendedMembers get(fn suspended_member): map T::AccountId => Option<T::BlockNumber>;
+		pub SuspendedMembers get(fn suspended_member): linked_map T::AccountId => Option<T::BlockNumber>;
```

Comment:
avoid linked_map - use blake2_128_concat or twox64_concat (if the `AccountId` only comes from a signed origin).